pipeline/assets/config.json

```diff
@@ -19,5 +19,7 @@
     "service_date": { "max": "NA", "min": "1990-01-01" }
   },
   "notif_django_path": "output/notif.txt",
-  "unobserved_stop_distance": 50
-}
+  "unobserved_stop_distance": 50,
+  "output_path": "output/csv/",
+  "output_type": "aperture"
+}
```

Comment:
I just wanna make sure but don't other parts of the application use output_path so setting it to output/csv/ could be an issue?

Reply:
I think output_path is used only when saving to csv. At least that's what I intended, idk if there are other uses.